src/lightning/pytorch/loggers/neptune.py

```diff
@@ -48,6 +49,23 @@
 _INTEGRATION_VERSION_KEY = "source_code/integrations/pytorch-lightning"
 
 
+def catch_inactive(func: Callable) -> Callable:
```

Comment:
```suggestion
def _catch_inactive(func: Callable) -> Callable:
```
Since the user is probably not meant to import this, I suggest to mark it protected

Reply:
You're right. Done